core/platforms/packaging/bareos.spec

```diff
@@ -708,29 +706,27 @@ Requires: php-xmlreader
 Requires: php-xmlwriter
 Requires: php-zip
 
-%if 0%{?suse_version} || 0%{?sle_version}
-BuildRequires: apache2
-# /usr/sbin/apxs2
-BuildRequires: apache2-devel
-BuildRequires: mod_php_any
-%define _apache_conf_dir /etc/apache2/conf.d/
-%define www_daemon_user  wwwrun
-%define www_daemon_group www
-Requires: apache
-Recommends: mod_php_any
-%else
-BuildRequires: httpd
-# apxs2
-BuildRequires: httpd-devel
+%if !0%{?suse_version}
+Requires: php-libxml
+%endif
+
+%if 0%{?centos_version} || 0%{?rhel_version} || 0%{?fedora}
+Requires: httpd
 %define _apache_conf_dir /etc/httpd/conf.d/
-%define www_daemon_user  apache
+%define www_daemon_user apache
 %define www_daemon_group apache
-%if 0%{?fedora_version} >= 33 || 0%{?rhel_version} >= 900
-Requires:   php-fpm
+%endif
+
+%if 0%{?suse_version} || 0%{?sle_version}
+%if 0%{?sle_version} == 120500
+Requires: apache2
```

Comment:
I don't understand why we restrict the webui to apache2, and force its installation. That defeat the use of 
Requires: httpd
This is provides by apache2, nginx, lighthttpd package so administrator are free to use the webserver they master.

Reply:
We've decided to keep the restriction to Apache2 for now.